### ISSUE TITLE
Error when overriding isSatisfied

### ISSUE THREAD

**Issue by Mighto360** [davecom/SwiftCSP#9]
When version 0.9.6 is installed for Swift 4.2 via CocoaPods, I'm getting this error on build:

`Overriding non-open instance method outside of its defining module`

I'm trying to create a Constraint subclass from **outside the module** as such (error on "`override func...`" line:

```
final class SampleConstraint: UnaryConstraint<String, String> {
    init(str: String) {
        super.init(variable: str)
    }
    override func isSatisfied(assignment: Dictionary<String, String>) -> Bool {
        if (something) {
            return true
        } else {
            return false
        }
    }
}
```

 It appears that because isSatisfied is defined as public, I can't override it. Does the source just need to be changed to make this method open, or is this my fault?

**Comment by Mighto360** [comment]
I managed to get rid of the build error by simply including the source files in my project instead of installing via CocoaPods (that way, it's all within the same module). However, I'm not going to close it yet because this should probably be fixed especially if the README says that CocoaPod installation is fine.

**Comment by davecom** [comment]
Hi, @Mighto360. You're absolutely right. It should be marked as `open`. I'll make a fix and post a new version to CocoaPods (0.9.8). Thanks for the report!

**Comment by davecom** [comment]
Closed by 04bcffcc7aa67cd4952cbf07610709e9852bc9fa
